scss/forms/_form-check.scss

```diff
@@ -100,10 +99,12 @@
     pointer-events: none;
     background-color: $input-disabled-bg; // Boosted mod
     filter: none;
+    border-color: $gray-500; // Boosted mod
```

Comment:
The border color isn't already `$gray-500` when it's not disabled?

Reply:
It is, but we override it with `:active` in Boosted (Bs only apply a filter on checkbox) which leads to an orange border when the label is clicked. I tried another one: I've added a `pointer-events: none` on the label following the input.